### PR TITLE
Css three column layout-Bo Mercado

### PR DESCRIPTION
**Pull request by rbomercado**
image

### CODE REVIEW

File: css-three-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          existing tolls,such as JIRA,GitHub
+          and Jenkins with Enzyme</p>
+      </div>
+      <div class="column-one-third">
+        <p class="font-size-smallest light-grey-text">We train your entire team and provide
+          you with support when and as its
+          needed</p>
+      </div>
+    </div>
+  </div>
```
Comment (codingcodymiller) [inline css-three-column-layout/index.html:59]: I can see why you would do this with layout classes in mind, but by separating each chunk out to its own row, now the content looks like it isn't related, which it very much is. You should have one row with three columns, and each column holds the image, header, and text for each chunk.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-three-column-layout/index.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    <div class="text-align-center ">
+      <div class="row">
+        <div class="column-full">
+          <header class=" font-size-bigger white-text"> Designed for life science.Optimized for SaMD and Digital Health.
+          </header>
+        </div>
+      </div>
+
+      <div class="column">
+        <div class="column-one-thirds">
```
Comment (codingcodymiller) [inline css-three-column-layout/index.html:23]: `column` and `column-one-thirds` at once? You should only need one `column` element, not a `column` as a child of a `column`.

File: css-three-column-layout/index.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                <img class="size-pic" src="images/discussion.png" alt="discussion">
+              </div>
+              <div>
+                <p class="white-text">In-depth Onboarding</p>
+              </div>
+              <div>
+                <p class="font-size-smallest light-grey-text">
+                  We train your entire team and provide
+      you with support when and as its
+      needed</p>
```
Comment (codingcodymiller) [inline css-three-column-layout/index.html:66]: Your indentation seems a bit out there, please be very careful to keep your formatting correct.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-three-column-layout/index.html (hunk 2 above)
Comment (codingcodymiller) [reply css-three-column-layout/index.html:25]: This note still needs to be addressed. The three `column-one-thirds` elements here should be children of an element with the `row` class. `column` class elements must always be a direct child of an element with the class `row`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by rbomercado** [comment]
revamped the css and html 
image

File: css-three-column-layout/index.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <div class="container align-center text-align-center">
+    <div class="header">
+      <header class=" white-text-big"> Designed for life science.Optimized for SaMD and Digital Health.</header>
+    </div>
+    <div class="row align-center">
+      <div class="column">
```
Comment (codingcodymiller) [inline css-three-column-layout/index.html:20]: `column` what? How big is this column? `column` classes should be named after their size. This one is a third, so it should be `column-third`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-three-column-layout/index.html (hunk 4 above)
Comment (rbomercado) [reply css-three-column-layout/index.html]: More descriptive! updated css/html from column to column-third.

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**